sdk/communication/communication-short-codes/test/public/utils/testUSProgramBrief.ts

```diff
@@ -0,0 +1,73 @@
+
+import { USProgramBrief } from "../../../src";
+
+export function getTestUSProgramBrief(id: string) {
```

Comment:
You can just give `id` a default value here so you don't have to do the check you do below. Something like:

 `...(id: string = "<default-id>")`